active-rfcs/0041-reactivity-effect-scope.md

```diff
@@ -161,17 +161,17 @@ parentScope.run(() => {
   // the scope will not be collected and disposed by the outer scope
   childScope = effectScope(true /* detached */)
   childScope.run(() => {
-    watch(doubled, () => console.log(double.value))
+    watch(doubled, () => console.log(doubled.value))
   })
 
-  watchEffect(() => console.log('Count: ', double.value))
+  watchEffect(() => console.log('Count: ', doubled.value))
 })
 
 // disposes all effects, but not `childScope`
 parentScope.stop()
 
-// stop the nested scope only when appropriate
-nestedScope.stop()
+// stop the child scope only when appropriate
```

Comment:
Can you adapt it to `nestedScope` instead?

Reply:
@posva Sure, i changed it. Can you check again?